DatadogWebViewTracking/Tests/WKUserContentController+DatadogTests.swift

```diff
@@ -144,71 +142,129 @@ class WKUserContentController_DatadogTests: XCTestCase {
     }
 
     func testSendingWebEvents() throws {
-        let core = DatadogCoreProxy(
-            context: .mockWith(
-                service: "default-service-name",
-                env: "tests",
-                version: "1.0.0",
-                applicationBundleIdentifier: "com.datadoghq.ios-sdk",
-                featuresAttributes: [
-                    "rum": [
-                        "ids": [
-                            RUMContextAttributes.IDs.sessionID: UUID.nullUUID.uuidString.lowercased(),
-                            RUMContextAttributes.IDs.applicationID: String.mockAny()
-                        ]
-                    ]
-                ]
-            )
+        let logMessageExpectation = expectation(description: "Log message received")
+        let rumMessageExpectation = expectation(description: "RUM message received")
+        let core = PassthroughCoreMock(
+            messageReceiver: FeatureMessageReceiverMock { message in
+                switch message {
+                case .custom(key: let key, baggage: let baggage):
+                    switch key {
+                    case "browser-log":
+                        let event = baggage.attributes as JSON
+                        XCTAssertEqual(event["date"] as! Int64, 1_635_932_927_012)
+                        XCTAssertEqual(event["message"] as! String, "console error: error")
+                        XCTAssertEqual(event["status"] as! String, "error")
+                        XCTAssertEqual(event["view"] as! [String: String], ["referrer": "", "url": "https://datadoghq.dev/browser-sdk-test-playground"])
+                        XCTAssertEqual(event["error"] as! [String : String], ["origin": "console"])
+                        XCTAssertEqual(event["session_id"] as! String, "0110cab4-7471-480e-aa4e-7ce039ced355")
+                        logMessageExpectation.fulfill()
+                    case "browser-rum-event":
+                        let event = baggage.attributes as JSON
+                        XCTAssertEqual((event["view"] as! JSON)["id"] as! String, "64308fd4-83f9-48cb-b3e1-1e91f6721230")
```

Comment:
We can avoid forcing the unwrap here, I think `XCTAssertEqual` accepts optional.